modules/service/trac-price-service.js

```diff
@@ -1,47 +1,25 @@
 const axios = require('axios');
 const constants = require('../constants');
 
-const coinGeckoLink = 'https://api.coingecko.com/api/v3/simple/price?ids=origintrail&vs_currencies=eth';
+const coinGeckoLink = '';
 
 class TracPriceService {
     constructor(ctx) {
         this.logger = ctx.logger;
         this.config = ctx.config;
     }
 
-    async getTracPriceInEth() {
-        if (process.env.NODE_ENV === 'development') {
-            this.logger.trace(`Using default trac price in eth from configuration: ${this.config.blockchain.trac_price_in_eth}`);
-            return this.config.blockchain.trac_price_in_eth;
-        }
 
-        const now = new Date().getTime();
-        if (this.config.blockchain.trac_price_in_eth_last_update_timestamp
-            + constants.TRAC_PRICE_IN_ETH_VALIDITY_TIME_IN_MILLS > now) {
-            this.logger.trace(`Using trac price in eth from configuration: ${this.config.blockchain.trac_price_in_eth}`);
-            return this.config.blockchain.trac_price_in_eth;
-        }
-        let tracPriceInEth = this.config.blockchain.trac_price_in_eth;
-        const response = await axios.get(coinGeckoLink)
+    async getTracPrice() {
+        const response = await axios.get('https://api.coingecko.com/api/v3/simple/price?ids=origintrail&vs_currencies=eth')
```

Comment:
I recommend we avoid hard-coding like this, or if we have to, the previous implementation with the hardcoded value up top is cleaner.